content/course-specific-material/introR-NDWSC-Nov17.Rmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "ND WSC Intro to R Training"
+date: "2017-11-21"
```

Comment:
I normally put the date of the actual training - doesn't matter too much, but it keeps them in order.